src/components/Header.js

```diff
@@ -2,13 +2,17 @@ import { Link } from 'react-router-dom';
 
 const Header = () => (
   <nav className="navbar">
-    <h1>Bookstore CMS</h1>
+    <h1 className="nav-header">Bookstore CMS</h1>
     <ul>
```

Comment:
- [ ] Kindly it will be better 😃 if you put the correct color for the text here to make it nice like the original design as you see here it is different from the original design, so let's make it like the zeplin design 💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|   ![image](https://github.com/Jeddaa/bookstore/assets/83360636/ca49eb62-b639-4998-aaa2-d92daff59eb2) |   ![image](https://github.com/Jeddaa/bookstore/assets/83360636/b12a0030-fde5-4455-a789-9b8ccd783cb5)  |